src/WebUI/Program.cs

```diff
@@ -22,6 +23,7 @@
 builder.Services.AddSingleton<MarkdigPipelineService>();
 builder.Services.AddScoped<SswRulesGptDialogService>();
 builder.Services.AddScoped<ApiKeyValidationService>();
+builder.Services.AddGoogleAnalytics("G-239HQG61FX");
```

Comment:
Can we move this to a config variable?

Reply:
@Harry-Ross I've moved the ID into appsettings